index.bs

```diff
@@ -14589,7 +14589,8 @@ must propagate to the caller, and if
 not caught there, to its caller, and so on.
 
 Per [[#conventions]], an algorithm specified in this document may intercept thrown exceptions, either by specifying
-the exact steps to take if <dfn>an exception was thrown</dfn>, or by explicitly handling [=abrupt completions=].
+the exact steps to take if <dfn export>an exception was thrown</dfn>, or by explicitly handling
+[=abrupt completions=].
```

Comment:
You'll have to change more of the wording here if you want this to be generally useful ("in this document" makes it not work).

However, we also define this in https://infra.spec.whatwg.org/#algorithm-control-flow and I suspect we want to centralize on that.